src/cpp/server/secure_server_credentials.cc

```diff
@@ -148,9 +148,9 @@ std::shared_ptr<ServerCredentials> LocalServerCredentials(
 }
 
 std::shared_ptr<ServerCredentials> TlsServerCredentials(
-    const TlsCredentialsOptions& options) {
+    const std::shared_ptr<TlsCredentialsOptions>& options) {
```

Comment:
Same question as above w.r.t `shared_ptr`.

Reply:
Changed to constant reference.